packages/ds-mui-theme/src/components/SearchableSelect.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
```

Comment:
// Globaly, this file contain a lot of different components. It would be great to only have one component per file.